documentation/Semantic-and-Model-Violations-Reference.md

```diff
@@ -611,6 +622,14 @@ This document lists the set of automated rules that can be validated against swa
 
 **How to fix the violation**: Provide the requried property in example or remove this property from the required list of the definition in swagger spec. Or add items property if the type is array.
 
+### <a name="DISCRIMINATOR_VALUE_NOT_FOUND" />DISCRIMINATOR_VALUE_NOT_FOUND
+
+**Output Message**: Discriminator value "{0}" not found
+
+**Description**: The property used as discriminator has a value {0} but it's not found in swagger spec.
+
+**How to fix the violation**: Add the model that has the discriminator value. The discriminator value could be specified by model name in definitions or by "x-ms-discriminator-value".
```

Comment:
Regard to `how to fix`, should tell user to correct the discriminator value.

Reply:
Updated